Makefile

```diff
@@ -209,6 +209,38 @@ check-kuttl:
 	${PGO_KUBE_CLIENT} ${KUTTL_TEST} \
 		--config testing/kuttl/kuttl-test.yaml
 
+# KUTTL_ params -- could be left out entirely and just set as env vars
+KUTTL_PSQL_VERSION ?= 14
+# TODO: should the test image here be set according to the KUTTL_PSQL_VERSION
+KUTTL_PSQL_IMAGE ?= registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-14.1-0
+
+# TODO: When I attempted to loop through the directories and find the yaml files,
+# due to how makefile expands variables, each file would end up in each directory.
+# So this implementation loops through to create the folders under generated
+# and then loops through all the yaml files, using the dirname/basename to place
+# the generated file in the correct folder
```

Comment:
🤔 It definitely feels like Make built-ins could be used to get there, but I don't know it well enough to suggest a solution.

I don't think we need separate directories for separate versions if that helps. I think we can probably get by with a single source directory and a single target directory. When we want to test a different version, we invoke `make` again with different arguments. Like,

```shell
make generate-kuttl check-kuttl PG_VERSION=10
make generate-kuttl check-kuttl PG_VERSION=11
```

If that is the case, then patsubst or something might work.
`testing/kuttl/source/*/*.yaml --> testing/kuttl/dest/*/*.yaml`

Reply:
I was thinking our procedure would be 
```
generate 13
generate 14
test 13 + 14
```
but I am curious about the idea of our CI running something more like
```
test 13 (generates, then tests)
test 14 (generates, then tests)
```
The only slight disadvantage that I see here is around what behavior we want to see when a single test in `test 13` fails. Like, if I run KUTTL on 13 + 14, I'll get all the failures/successes -- what behavior do we want to see if we run one test after the other? That's probably something we can discuss and tweak in our CI, i.e., we probably want the failures to log and then to go on to the `test 14` command so that we can see a full report of successful/failed tests after they all run.

That said, I'm looking at this make command now and really wanting to simplify it a bit, so let me follow this thread a little.

Reply:
8 hours later, this Make target looks cleaner, but also I am tired. (I mean, I did also make sure a 4yo was fed, bathed, and slept, so it wasn't constant 8 hours, but still: tired.)

Reply:
> The only slight disadvantage that I see here is around what behavior we want to see when a single test in `test 13` fails. Like, if I run KUTTL on 13 + 14, I'll get all the failures/successes -- what behavior do we want to see if we run one test after the other? That's probably something we can discuss and tweak in our CI

Yeah, we can leave that to something else. It can run the two versions in a single job, separate jobs, sequentially, or in parallel, etc.